lib/components/Button.js

```diff
@@ -22,10 +22,17 @@ const Button = props => {
     dataTestId,
     size,
     href,
+    focus,
```

Comment:
We don't need the focus prop

Reply:
@goutham-subramanyam 

If i'm understanding correctly, then we can remove this section too if we remove `focus` attr right:
https://github.com/bigbinary/neeto-help-web/blob/e962f3766ccb579b2f731035067ce57c819510c2/app/javascript/Admin/Components/NitroUI/Button.js#L32-L34

Reply:
Yes